### ISSUE TITLE
Unsupported locale error at fresh install

### ISSUE THREAD

**Issue by Cediddi** [giantpinkrobots/flatsweep#10]
I've just installed the app from flathub, I ran it with  
`$ flatpak run io.github.giantpinkrobots.flatsweep`  
but I'm getting this as output:
```pytb
Traceback (most recent call last):
  File "/app/bin/flatsweep", line 27, in <module>
    from flatsweep import main
  File "/app/share/flatsweep/flatsweep/main.py", line 19, in <module>
    locale.setlocale(locale.LC_ALL, os.getenv("LANG"))
  File "/usr/lib/python3.10/locale.py", line 620, in setlocale
    return _setlocale(category, locale)
locale.Error: unsupported locale setting
```

my locale looks like this:
```env
LANG=en_DE.UTF-8
LANGUAGE=
LC_CTYPE="en_DE.UTF-8"
LC_NUMERIC=en_DE.UTF-8
LC_TIME=en_DE.UTF-8
LC_COLLATE="en_DE.UTF-8"
LC_MONETARY=en_DE.UTF-8
LC_MESSAGES="en_DE.UTF-8"
LC_PAPER=de_DE.UTF-8
LC_NAME=de_DE.UTF-8
LC_ADDRESS=de_DE.UTF-8
LC_TELEPHONE=de_DE.UTF-8
LC_MEASUREMENT=en_DE.UTF-8
LC_IDENTIFICATION=de_DE.UTF-8
LC_ALL=
```
I'm using Kubuntu 22.04.3 and Flatpak 1.12.7

**Comment by giantpinkrobots** [comment]
Hello. It seems like the en_DE locale causes problems. To fix this I've added a try/except statement that defaults to United States English if there is an error. Can you confirm that it fixes the issue by installing the flatpak file you can find below? It's in that zip file.
[io.github.giantpinkrobots.flatsweep.zip](https://github.com/giantpinkrobots/flatsweep/files/12376547/io.github.giantpinkrobots.flatsweep.zip)

**Comment by giantpinkrobots** [comment]
Hello, deleted my previous comment. That version didn't work. I think I've fixed it now. The en_DE locale seems not to be supported here. Thus I added a try/except statement that will default to US English if there is an error. For now it should work.

[io.github.giantpinkrobots.flatsweep.zip](https://github.com/giantpinkrobots/flatsweep/files/12378726/io.github.giantpinkrobots.flatsweep.zip)

Can you check by installing this directly?

**Comment by giantpinkrobots** [comment]
This issue should be fixed with the v2023.8.21 version. Please reopen if you still encounter issues, but for now I am closing this as completed.
